Readme.md

```diff
@@ -39,8 +39,8 @@ a Riemannian manifold.
 
 The following packages are related to `Manopt.jl`
 
-* [`Manifolds.jl`](https://juliamanifolds.github.io/Manifolds.jl/stable/) – a library of manifolds implemented using [`ManifoldsBase.jl`](https://juliamanifolds.github.io/ManifoldsBase.jl/stable/) :octocat: [GitHub repository](https://github.com/JuliaManifolds/Manifolds.jl)
```

Comment:
This was an en-dash, but it should be an em-dash, and typically no spaces are used surrounding an em-dash.

Reply:
I am very much used to spaces in the languages I use besides English, so this really looks horrible now I think. Then I would prefer just a space instead I think.

Reply:
It seems spaces are sometimes used
https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use#:~:text=Spacing%20around%20an%20em%20dash,right%20up%20next%20to%20it.

But perhaps an em-dash is the wrong choice. What about just a comma?

Reply:
a comma would also be ok, I think.